targets/zephyr/Makefile.zephyr

```diff
@@ -47,6 +47,11 @@ endif
 
 INTERM     = build/$(BOARD)/obj-$(BOARD)
 OUTPUT     = build/$(BOARD)
+DOTCONFIG  = $(OUTPUT)/zephyr/.config
+
+include $(DOTCONFIG)
```

Comment:
These patches do not introduce any change to the build process (from user's point of view).

An include file is an implicit build target and, because there is a rule that tells make how to generate $(DOTCONFIG), it can be derived automatically.